app-utils/chafa/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=chafa
+PKGDES="A image-to-text converter for terminal"
+PKGSEC=utils
+PKGDEP="bash"
+BUILDDEP="libwebp freetype libxml2 libjpeg-turbo librsvg libjxl libtiff libavif libxslt libtool glib gtk-doc imagemagick"
```

Comment:
Okay, I think most of the build-time dependencies are actually runtime ones:

```
mingcongbai@JellyNote [ ~ ] $ readelf -d /usr/bin/chafa

Dynamic section at offset 0x2c538 contains 41 entries:
  标记        类型                         名称/值
 0x0000000000000001 (NEEDED)             共享库：[libjpeg.so.8]
 0x0000000000000001 (NEEDED)             共享库：[librsvg-2.so.2]
 0x0000000000000001 (NEEDED)             共享库：[libgobject-2.0.so.0]
 0x0000000000000001 (NEEDED)             共享库：[libcairo.so.2]
 0x0000000000000001 (NEEDED)             共享库：[libtiff.so.5]
 0x0000000000000001 (NEEDED)             共享库：[libwebp.so.7]
 0x0000000000000001 (NEEDED)             共享库：[libwebpdemux.so.2]
 0x0000000000000001 (NEEDED)             共享库：[libjxl.so.0.10]
 0x0000000000000001 (NEEDED)             共享库：[libjxl_threads.so.0.10]
 0x0000000000000001 (NEEDED)             共享库：[libavif.so.15]
 0x0000000000000001 (NEEDED)             共享库：[libfreetype.so.6]
 0x0000000000000001 (NEEDED)             共享库：[libchafa.so.0]
 0x0000000000000001 (NEEDED)             共享库：[libglib-2.0.so.0]
 0x0000000000000001 (NEEDED)             共享库：[libc.so.6]
```

So you should add these to PKGDEP:

```
cairo freetype glib libavif libjpeg-turbo libjxl librsvg libtiff libwebp
```

Reply:
Added, thanks!